libs/features/creature/src/creature-text/creature-text.integration.spec.ts

```diff
@@ -0,0 +1,280 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { MysqlQueryService, SqliteService } from '@keira/shared/db-layer';
+import { MultiRowEditorPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { CreatureText } from '@keira/shared/acore-world-model';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { ToastrModule } from 'ngx-toastr';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { of } from 'rxjs';
+import { CreatureHandlerService } from '../creature-handler.service';
+import { SaiCreatureHandlerService } from '../sai-creature-handler.service';
+import { CreatureTextComponent } from './creature-text.component';
+import Spy = jasmine.Spy;
+import { instance, mock } from 'ts-mockito';
+class CreatureTextPage extends MultiRowEditorPageObject<CreatureTextComponent> {}
+
+describe('CreatureText integration tests', () => {
+  let fixture: ComponentFixture<CreatureTextComponent>;
+  let queryService: MysqlQueryService;
+  let querySpy: Spy;
+  let handlerService: CreatureHandlerService;
+  let page: CreatureTextPage;
```

Comment:
I'd prefer if we avoided shared-variables across tests and instantiate and return them in the `setup` function.

For example: https://github.com/azerothcore/Keira3/blob/master/libs/features/creature/src/creature-template-spell/creature-template-spell.integration.spec.ts#L33